src/hip/batched_transpose_sol.cpp

```diff
@@ -322,7 +322,11 @@ BatchedTransposeSolution::BatchedTransposeSolution(const ExecutionContext& ctx,
                                                    uint32_t width_)
     : data_type(data_type_), batch(batch_), height(height_), width(width_)
 {
-    if(data_type == miopenInt8x4 || data_type == miopenDouble)
+    if(!(data_type == miopenHalf     //
+         || data_type == miopenFloat //
+         || data_type == miopenInt32 //
+         || data_type == miopenInt8  //
+         || data_type == miopenBFloat16))
```

Comment:
Are you saying some test fails? AFA I understand, F8 can use the same kernels as int8 because we only transpose values and we don't change them with any mathematical operations.

Reply:
@amberhassaan 
> Are you saying some test fails?

No. This is impossible because we do not have F8 testing for BN.

> F8 can use the same kernels as int8 because we only transpose values and we don't change them with any mathematical operations

Theoretically yes, but we need to check the code (by eyes at least) and run some testing at least before enabling this.

Note that #2251 explicitly says that "Only convolution operations are supported."

Reply:
Understood. I can take up batch_transpose support for F8 if needed in the future.